components/Vote/ElectionConfirm.js

```diff
@@ -0,0 +1,345 @@
+      <Chart
+        config={{
+          type: 'Bar',
+          numberFormat: '.0%',
+          color: 'key',
+          colorMap: {
+            weiblich: '#9467bd',
+            ['nichtbinär']: 'neutral',
+            ['männlich']: '#2ca02c'
+          },
+          colorSort: 'none',
+          colorLegend: true,
+          colorLegendValues: ['weiblich', 'nichtbinär', 'männlich'],
+          domain: [0, 1],
+          sort: 'none',
+          inlineValue: true
+        }}
+        values={values.filter(v => v.value !== '0')}
+      />
+    </div>
+  )
+})
+
+const getAge = birthYear => new Date().getFullYear() - birthYear
+
+const getLabel = age =>
+  age < 30
+    ? '< 30'
+    : age < 40
+    ? '30-40'
+    : age < 50
+    ? '40-50'
+    : age < 60
+    ? '50-60'
+    : '> 60 Jahre'
+
+const CandidatesAge = voteT(({ candidates, vt }) => {
+  const candidatesWithBirthday = candidates.filter(
+    candidate => candidate.yearOfBirth
+  )
+  const values = candidatesWithBirthday
+    .reduce(
+      (acc, candidate) =>
+        acc.map(item =>
+          item.key === getLabel(getAge(candidate.yearOfBirth))
+            ? { ...item, value: item.value + 1 }
+            : item
+        ),
+      [
+        { key: '< 30', value: 0 },
+        { key: '30-40', value: 0 },
+        { key: '40-50', value: 0 },
+        { key: '50-60', value: 0 },
+        { key: '> 60 Jahre', value: 0 }
+      ]
+    )
+    .map(getPercentString(candidatesWithBirthday.length))
+
+  return (
+    <div {...styles.chart}>
+      <Label>{vt('vote/election/confirm/age/header')}</Label>
+      <Chart
+        config={{
+          type: 'Bar',
+          numberFormat: '.0%',
+          color: 'key',
+          colorMap: {
+            ['< 30']: '#1f77b4',
+            ['30-40']: '#ff7f0e',
+            ['40-50']: '#2ca02c',
+            ['50-60']: '#d62728',
+            ['> 60 Jahre']: '#9467bd'
+          },
+          colorSort: 'none',
+          colorLegend: true,
+          colorLegendValues: ['< 30', '30-40', '40-50', '50-60', '> 60 Jahre'],
+          domain: [0, 1],
+          sort: 'none',
+          inlineValue: true
+        }}
+        values={values.filter(v => v.value !== '0')}
+      />
+    </div>
+  )
+})
+
+const ElectionConfirm = compose(
+  voteT,
+  graphql(submitElectionBallotMutation, {
+    props: ({ mutate }) => ({
+      submitElectionBallot: (electionId, candidacyIds) => {
+        return mutate({
+          variables: {
+            electionId,
+            candidacyIds
+          }
+        })
+      }
+    })
+  })
+)(({ election, vote, submitElectionBallot, goBack, vt }) => {
+  const [isUpdating, setUpdating] = useState(false)
+  const [error, setError] = useState(null)
+  const ref = useRef()
+  const [headerHeight] = useHeaderHeight()
+
+  useEffect(() => {
+    const { top } = ref.current.getBoundingClientRect()
+    const { pageYOffset } = window
+    const target = pageYOffset + top - headerHeight - 80
+    scrollIt(target, 100)
```

Comment:
IMO just jump immediately, the animation just makes it feel wonky since the page is jump resizing anyways (list disappearing)
```suggestion
    window.scroll(0, target)
```